core/commands/swarm.go

```diff
@@ -61,6 +62,140 @@ const (
 	swarmDirectionOptionName = "direction"
 )
 
+var swarmPeeringCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "modify the peering service.",
```

Comment:
```suggestion
		Tagline: "Manage the peering subsystem.",
```

- We refer to it as the peering "subsystem" in the docs.
- Capitalize.

Reply:
Comment applies to other commands as well.

Reply:
Fixed.